src/NHibernate.Test/Legacy/MultiTableTest.cs

```diff
@@ -207,11 +206,6 @@ public void Constraints()
 		[Test]
 		public void MultiTable()
 		{
-			if (Dialect is AbstractHanaDialect)
-			{
-				Assert.Ignore("feature not supported: Currently specify table name by 'FOR UPDATE of t1.c1' if there are more than one tables/views/subqueries in the FROM clause");
-			}
```

Comment:
Removing this does not cause any failure in my tests. Was it failing in some earlier version of the dialect, but you would have fix that then forgot about the test change here?

Reply:
This is new with HANA 2 SP3. I wrote the dialect for HANA 2 SP2 where it's failing. But I think it's ok to require a minimum of HANA 2 SP3 for NHibernate.

Reply:
Well, considering this does not change anything in the NHibernate assembly, this will be a requirement only for the test assembly. So yes, let the tests have this requirement.